angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.module.ts

```diff
@@ -23,6 +18,7 @@ import {
     RouterModule,
     SprkMastheadAccordionModule,
     SprkMastheadAccordionItemModule,
+    SprkMastheadSelectorModule,
```

Comment:
we'll wanna add SprkMastheadSelector to the exports too, to ensure it gets received by apps importing on one on one basis